lib/apiservers/portlayer/restapi/handlers/storage_handlers_test.go

```diff
@@ -142,6 +142,10 @@ func (c *MockDataStore) CreateImageStore(op trace.Operation, storeName string) (
 	return u, nil
 }
 
+func (c *MockDataStore) DeleteImageStore(op trace.Operation, storeName string) error {
```

Comment:
FYI: this mock object is a very good candidate to be generated rather than implemented.